common-util/constants.js

```diff
@@ -21,10 +21,13 @@ export const CORE_TECHNICAL_DOCUMENT =
 export const FLIPSIDE_URL =
   'https://flipsidecrypto.xyz/flipsideteam/olas-key-activity-metrics-pnPjda';
 export const DUNE_QUERY_URL = 'https://dune.com/queries';
-export const PREDICTION_DAA_QUERY_ID = '4165113';
+export const PREDICTION_DAA_QUERY_ID = '4185253';
```

Comment:
was this changed accidentally? the value looks the same as mech query's, but I think should be the old one, the metric is also broken in Vercel

Reply:
Yes, I think it was changed on accident. I'll change this in a new PR real quick